### PR TITLE
[FIX] Disabled spaces are listed

### PR DESCRIPTION
**Pull request by joragua**
## Related Issues
App: https://github.com/owncloud/android/issues/4405

- [x] Add changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
- [x] Add feature to Release Notes in `ReleaseNotesViewModel.kt` creating a new `ReleaseNote()` with String resources (if required)

_____

### CODE REVIEW

File: changelog/unreleased/4406
Comment (JuancaG05) [inline changelog/unreleased/4406:1]: This is regressive so we don't need a calens here 😉

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Nice! We just need to remove calens here @joragua 🚀

File: changelog/unreleased/4406
Comment (joragua) [reply changelog/unreleased/4406:1]: Done ✅

**Review by JuancaG05 — APPROVED** [review]
LGTM!!

**Comment by jesmrec** [comment]
Checked in the three main cases for spaces listing:

- List of spaces
- Folder picker in oCIS for copy operation
- Sharing from 3rd party app to oCIS account

In every of them, checking listing and filtering with some disabled spaced.

That's OK from my side

**Pull request merged**